### PR TITLE
Symfony 5

### PR DESCRIPTION
**Pull request by Sam-Burns**
Allow Behat to test Symfony 5 console applications

### CODE REVIEW

**Comment by Sam-Burns** [comment]
Needs a release tag on `behat/transliterator`, as discussed here: https://github.com/Behat/Transliterator/issues/25

File: composer.json [hunk 1]
```diff
@@ -17,13 +17,13 @@
         "php": ">=5.3.3",
         "ext-mbstring": "*",
         "behat/gherkin": "^4.6.0",
-        "behat/transliterator": "^1.2",
-        "symfony/console": "~2.7.40||^2.8.33||~3.3.15||^3.4.3||^4.0.3",
-        "symfony/config": "~2.3||~3.0||~4.0",
-        "symfony/dependency-injection": "~2.1||~3.0||~4.0",
+        "behat/transliterator": "dev-master#41fa9aca9838d139a646920670c1c8f7e829e751",
+        "symfony/console": "~2.7.40||^2.8.33||~3.3.15||^3.4.3||^4.0.3||^5.0",
```
Comment (lzakrzewski) [inline composer.json]: will symfony dependencies work with just this: `~4.0||~5.0`

Comment (Sam-Burns) [reply composer.json]: I guess, but not in PHP 7.4. Do you think I should remove the transliterator changes now?

File: composer.json [hunk 2]
```diff
@@ -17,13 +17,13 @@
...
+        "symfony/config": "~2.3||~3.0||~4.0||^5.0",
+        "symfony/dependency-injection": "~2.1||~3.0||~4.0||^5.0",
         "symfony/event-dispatcher": "~2.1||~3.0||~4.0",
         "symfony/translation": "~2.3||~3.0||~4.0",
```
Comment (snapshotpl) [inline composer.json]: What about this symfony stuff?

**Comment by Nyholm** [comment]
This is a duplicate of https://github.com/Behat/Behat/pull/1256

**Comment by ciaranmcnulty** [comment]
I've tagged transliterator and merged #1256 - can you please rebase and see if these changes are still needed?

Thanks for the effort and apologies for the slow feedback

**Pull request closed without merging**